src/components/buttons/ActionContent.tsx

```diff
@@ -0,0 +1,53 @@
+import { ReactNode } from "react";
+import { View, Text, StyleSheet } from "react-native";
+
+import Badge from "../badge/Badge";
+import Paragraph from "../typography/Paragraph";
+
+import { global } from "@/constants/DesignTokens";
+
+interface WalletContentProps {
+  icon: ReactNode;
+  title: string;
+  description: string;
+}
```

Comment:
Some places in the codebase are using `type` and other are using `interface`, you should come with a pattern for the app so it's the same for the entire application. My personal preference is `type`

```suggestion
type WalletContentProps = {
  icon: ReactNode;
  title: string;
  description: string;
}
```